server/client/controller.ts

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+import { resolve, sep } from 'path';
+import { existsSync, readdirSync } from 'fs';
+
+// function to recursively get all files from a directory
+const getFilesInDirectory: (directory: string) => Array<string> = (directory) =>
```

Comment:
I am not sure changing this at this time gives us any additional benefit. The content it will recurse is what we build (which should be 'simple' - no symlinks, a moderate number of files), and is not public (so cannot be used as a utility. If there was need of something similar in other code than absolutley makes sense to harden this/use 3rd party.